src/compiler/emitter.ts

```diff
@@ -2099,6 +2108,11 @@ module ts {
                     name = "_" + (tempCount < 25 ? String.fromCharCode(tempCount + (tempCount < 8 ? 0 : 1) + CharacterCodes.a) : tempCount - 25);

```

Comment:
Can you please add a comment `// Note that _i is skipped`, since the above is super subtle.